cmd/sealer/cmd/cluster/run.go

```diff
@@ -168,9 +179,10 @@ func runWithClusterfile(clusterFile string, runFlags *types.RunFlags) error {
 		}
 
 		return appInstaller.Install(imageName, AppInstallOptions{
-			Envs:        runFlags.CustomEnv,
-			RunMode:     runFlags.Mode,
-			IgnoreCache: runFlags.IgnoreCache,
+			Envs:            runFlags.CustomEnv,
+			RunMode:         runFlags.Mode,
+			IgnoreCache:     runFlags.IgnoreCache,
+			P2PDistribution: p2p,
```

Comment:
How about being consistent here and also using enum value

Reply:
Sounds great. I've changed the corresponding parts to use an enum instead.